pkgs/misc/vscode-extensions/default.nix

```diff
@@ -389,6 +389,18 @@ let
         extractNuGet = callPackage ./python/extract-nuget.nix { };
       };
 
+      msjsdiag.debugger-for-chrome = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "debugger-for-chrome";
+          publisher = "msjsdiag";
+          version = "4.12.11";
+          sha256 = "sha256-9i3TgCFThnFF5ccwzS4ATj5c2Xoe/4tDFGv75jJxeQ4=";
+        };
+        meta = {
+          license = stdenv.lib.licenses.mit;
```

Comment:
```suggestion
          license = lib.licenses.mit;
```

Reply:
What should the commit name be for this change?

I'd love to amend the previous commit but the last time I tried to commit locally, it took 3 hours (on a fast internet connection w/ ssd) before failing

<img width="1920" alt="Screen Shot 2021-01-18 at 10 26 32" src="https://user-images.githubusercontent.com/10779424/104903401-da193f00-5977-11eb-8ffc-f8ed5ec71d2f.png">

Reply:
A minor suggestion to make it easier for folks to contribute in the future without commit history being a blocking issue - PRs can be squashed into a single commit pre-merge and have the commit message set.

![Kapture 2021-01-18 at 10 36 04](https://user-images.githubusercontent.com/10779424/104904555-316bdf00-5979-11eb-87af-5a8b26e24ffc.gif)

Reply:
Amending and force pushing should only take a few seconds. Also I know about this feature and certain people complain when I use it and I regularly forget to use it.

Reply:
>  Also I know about this feature and certain people complain when I use it and I regularly forget to use it.

I hear you dude!

It's taking me a while to clone the repo - it's gonna create some noise but I'll have to create new PRs for now until I get around to cloning the full history.